src/main/java/seedu/duke/Duke.java

```diff
@@ -2,13 +2,16 @@
 
 import seedu.command.Command;
 import seedu.command.CommandResult;
+import seedu.equipment.DuplicateSerialNumber;
```

Comment:
Oh I've been meaning to ask this, want to name this `DuplicateSerialNumberException`? So it's clearer

Reply:
Will do!